ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -35,6 +35,8 @@
 {% include 'eos/dhcp-servers.j2' %}
 {# switchport default #}
 {% include 'eos/switchport-default.j2' %}
+{# switchport port-security #}
```

Comment:
Please make sure the commands come somewhat it the right order on EOS. If you already checked this, just resolve this comment.

Reply:
yes